target/linux/ramips/dts/mt7621_mikrotik.dtsi

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
This file really shouldn't be called mt7621_mikrotik.dtsi

There are other MT7621 mikrotik devices (RBM11/RBM33) and they don't share all of the definitions of that file (GPIOS).

Most Routerboard devices share a commont platform though (especially the flash layout) so there might be room for a DTSI.

Reply:
Despite, my request was to create a _shared_ DTSI, which means that mt7621_mikrotik_rb750gr3.dts should also be based on that one. This is best done as a set of two commits:
1. Create a DTSI based on existing rb750gr3, i.e. move everything that your device uses as well into the DTS.
2. Add support for your device exploiting the DTSI.

> This file really shouldn't be called mt7621_mikrotik.dtsi

That's a valid remark. A better name could be something like mt7621_mikrotik_rb7xx.dtsi ...

Note that LEDs have to be in DTS files, as they contain part of the device name.

@f00b4r0 If I compare rbm33g and rb750gr3, there is not much in common, so I don't think a DTSI for "all" rb devices in mt7621 would be helpful (but rather make things more complicated).